sig-platform/README.md

```diff
@@ -1 +1,30 @@
-To be created by SIG...
\ No newline at end of file
+# ODH Platform Special Interest Group
+
+SIG Platform covers odh operator, core components, ecosystem integration, authetication, and monitoring  
+
+The [charter](./charter.md) defines the scope and governance of the Platform Special Interest Group.
+
+## Meetings
+[Monthly SIG meeting:] TBD
+[Meeting notes and agenda] RBD
+
+## Leadership
+
+### Chairs
+The Chairs of the SIG run operations and processes governing the SIG.
+- Taneem Ibrahim
+- Landon LaSmith
+- Samu Veloso Lopez
+- Gabe Goodheart
+
+### Members
+The members of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Chris Chase
+- JooHo Lee 
+- Sean Pryor 
+- Ben Herta 
+
+# Contact
+- [Slack](https://join.slack.com/t/odh-io/shared_invite/zt-18ptx7far-SWO4jkDbuA7Sq8Mut3JbcA)
```

Comment:
Can you  add the `#sig-platform` channel name to this? I think adding `...on #sig-platform` should work